### ISSUE TITLE
Error: $injector:modulerr Module Error

### ISSUE THREAD

**Issue by kmiloaguilar** [linnovate/mean#1486]
My inputs:
OS= OSX El Capitan
Node = 4.2.6
npm = 2.14.12
mean status = 
    MEAN Status
    -----------

    MEAN VERSION: 0.5.5

    DB connection successful!

    MongoDB URI: mongodb://localhost/mean-dev

    checking meanio and global mean-cli versions
    mean-cli at latest version: 0.11.1
    meanio at latest version: 0.8.83

So the gulp finish without any problems, and when I go to localhost:3000 it loads several files, even angularjs but I see the following error in the console.

Any thoughts? I did run `sudo npm install` and even runned the `postinstall` script even bower install too and problem persist.

```
Failed to instantiate module mean due to:
Error: [$injector:modulerr] http://errors.angularjs.org/1.5.0/$injector/modulerr?p0=m...)
    at Error (native)
    at http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:6:416
    at http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:40:60
    at n (http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:7:355)
    at g (http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:39:135)
    at http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:39:304
    at n (http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:7:355)
    at g (http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:39:135)
    at fb (http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:43:164)
    at Ac.c (http://localhost:3000/bower_components/angular/angular.min.js?v=8c9ac3aa67fee6aa599b4624398feba7:20:449
```

**Comment by ezequiel-umu** [comment]
Same here using Ubuntu 14.04, with fresh installation of nodejs and mean.

**Comment by rof20004** [comment]
Same here.

**Comment by arnason** [comment]
On same OS X,  seen it happening if you install some of the extra packages. Removing fixed it. Someone from the team can try replicating the issue by adding this one: "mean install mean-admin", then starting the server. To fix it I just removed it "mean uninstall mean-admin".

**Comment by ilyachenko** [comment]
Change "angular-ui-router": "#master" to "angular-ui-router": "latest" in bower.json
This solution helped me today.

**Comment by timelf123** [comment]
Confirmed, thanks for the fix @ilyachenko

**Comment by liorkesos** [comment]
This really helped me as well.
Good catch
Lior

On Tue, Mar 1, 2016 at 8:25 PM, Tim Elfelt <notifications@github.com> wrote:

> Confirmed, thanks for the fix @ilyachenko <https://github.com/ilyachenko>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linnovate/mean/issues/1486#issuecomment-190842175>.
>



-- 

Lior Kesos - http://www.linnovate.net
Linnovate - Community Infrastructure Care
mail: lior@linnovate.net
office: +972 722500881
cell: +972 524305252
skype: liorkesos

**Comment by liorkesos** [comment]
Fixed in f87b8f3a98082a041e0f24f80fd267e59dd35f44
